codice/ddf
# DDF-3181 Added ability to disable video thumbnail generation for large files

#### What does this PR do?
This PR adds a configuration ability to disable video thumbnail generation for large files.
#### Who is reviewing it?
@alexaabrd @josephthweatt @vinamartin @mcalcote 
#### Choose 2 committers to review/merge the PR.
@clockard
@figliold
@ricklarsen - Documentation
#### How should this be tested?
Follow the documentation to configure the Video Thumbnail Plugin, and confirm that only appropriate videos are processed and end up with thumbnails.
#### What are the relevant tickets?
[DDF-3181](https://codice.atlassian.net/browse/DDF-3181)
#### Screenshots (if appropriate)
![Video Thumbnail Plugin Configuration Modal](https://user-images.githubusercontent.com/8041246/28800841-7db1582a-7603-11e7-9e67-68d80a6039b7.png)
![Video Thumbnail Plugin Configuration Modal with Tooltip](https://user-images.githubusercontent.com/8041246/28800960-301d040a-7604-11e7-97f3-7e33d3d5458f.png)
![16.2.28. Video Thumbnail Plugin](https://user-images.githubusercontent.com/8041246/28801682-765aa1d6-7607-11e7-8ff7-686fc5331522.png)
#### Checklist:
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
